rainbows.rb

```diff
@@ -17,6 +19,14 @@
   [255,0,125] => "raspberry"
 }
 
+rainbow = [{:red => {r: 255, g: 0, b: 0}}, 
+          {:orange => {r: 255, g: 125, b: 0}}, 
+          {:yellow => {r: 255, g: 255, b: 0}}, 
+          {:green => {r: 0, g: 255, b: 0}}, 
+          {:blue => {r: 0, g: 0, b: 255}}, 
+          {:indigo => {r: 75, g: 0, b: 130}}, 
+          {:violet => {r: 125, g: 0, b: 255}}]
```

Comment:
nit-picky formatting. Put the []'s for the array on lines without hashes. This allows for better readability and easier deleting/commenting out of elements in the array.